packages/react/src/hooks/accounts/useDisconnect.test.ts

```diff
@@ -3,13 +3,18 @@ import { useAccount } from './useAccount'
 import { useConnect } from './useConnect'
 import { UseDisconnectConfig, useDisconnect } from './useDisconnect'
 
+function useTestAccount() {
+  const { ...values } = useAccount()
+  return values
+}
```

Comment:
Want to add `useTestAccount` to the `test/utils.ts` and export from `test/index.ts`?

Reply:
sg!